L2/beta_testnet_usage.md

```diff
@@ -0,0 +1,63 @@
+
+# Basic Info
+
+```bash
+Explorer：http://5.9.87.214/
+RPC: http://5.9.87.214:8545 
+Custom Gas Token: 0xe6ABD81D16a20606a661D4e075cdE5734AB62519
+Portal: 0xaa19a4faebbb8652988dee2a37c26aea9018ffeb
+System Config: 0x672bc488eb2e856f725e50afc2341bbe9a3eff30
+```
+
+# Get Custom Gas Token On L1
+
+First, ensure you've some sepolia gas, otherwise go [here](https://www.alchemy.com/faucets/ethereum-sepolia) for faucet.
+
+Then invoke the `drop` function on etherscan [here](https://sepolia.etherscan.io/address/0x274a6990dE7AaE06452cbEFa266c0C6a568F0D5B#writeContract).
+
+Or simply run this:
+```bash
+export L1_RPC_URL='http://88.99.30.186:8545'
+export PRIVATE_KEY=''# input your own pk
+
+cast send 0x274a6990dE7AaE06452cbEFa266c0C6a568F0D5B 'drop()' --private-key $PRIVATE_KEY -r $L1_RPC_URL
+```
+
+After that you can cross the claimed `Custom Gas Token` to L2 via `entrance` or follow the instructions [here](https://github.com/ethereum-optimism/specs/discussions/140#discussioncomment-9426636).
```

Comment:
Not clear about what `entrance` means here. Is it supposed with a link or something?

Reply:
It's a paid service, maybe @qzhodl will set it up later.